onnxruntime/core/session/custom_ops.cc

```diff
@@ -96,8 +96,17 @@ common::Status CreateCustomRegistry(const std::vector<OrtCustomOpDomain*>& op_do
   output = std::make_shared<CustomRegistry>();
 
   for (auto& domain : op_domains) {
-    if (domain->domain_[0])
-      ONNX_NAMESPACE::OpSchemaRegistry::DomainToVersionRange::Instance().AddDomainToVersion(domain->domain_, 1, 1000);
+    // Domain is not empty - add it to the DomainToVersion ONNX map
+    // If domain is empty, it is assumed to be part of the ONNX domain
+    if (domain->domain_[0]) {
+      // Add it to the DomainToVersion ONNX map if it doesn't already exist
+      // For example, two sessions using the same session_options should not add the same custom op domain to the version map twice
```

Comment:
Fails a debug assert in the ONNX code if we try to add an item that already exists in the map. hence, added this guard.